File: challenges/4-fizz-buzz/elixir/berutodo/fizzbuzz.exs
defmodule FizzBuzz do
  def receiveNumber(x) do
     1..x
     |> Enum.to_list()
     |> to_string_()
     |> Enum.map(&convert_and_evaluate_number/1)
  end
  defp to_string_(l) do
        l
        |> Enum.join(",")
        |> String.split(",")
    end

    defp convert_and_evaluate_number(elem) do
      elem |> String.to_integer() |> evaluate_numbers()
    end

    defp evaluate_numbers(number) when rem(number, 3) == 0 and rem(number, 5) == 0, do: :fizzbuzz
    defp evaluate_numbers(number) when rem(number, 5) == 0, do: :buzz
    defp evaluate_numbers(number) when rem(number, 3) == 0, do: :fizz
    defp evaluate_numbers(number), do: number
  end

File: challenges/4-fizz-buzz/python/marlonmartins2/fizz_buzz.py
n = int(input("Enter a number: "))
n2 = int(input("Enter a second number: "))

def fizzbuzz(n):
    response = ""
    if not n % 3:
        response += 'Fizz'
    if not n % 5:
        response += "Buzz"
    if not response:
        response = n
    return response


for i in range(1, n + 1):
    print(fizzbuzz(i))

print('\n')

for i in range(1, n2 + 1):
    print(fizzbuzz(i))